src/Components/Orders/index.js

```diff
@@ -222,8 +226,32 @@ export default class Orders extends Component {
         exporter.downloadCsv(rows);
       })
       .catch((error) => this.setState({downloading: false, error}));
+  };
+
+  handleSearchInput = (e: SyntheticEvent, el: {value: ?string}) => {
+    this.setState({search: el.value ? el.value : null});
+  };
+
+  handleSearchScope = (e: SyntheticEvent, el: {value: ?string}) => {
+    this.setState({searchScope: el.value})
   }
 
+  search = (e: SyntheticEvent) => {
+    const {search, searchScope, filters} = this.state;
+    let newFilters = filters.slice();
+    let filtersToRemove = this.searchOptions.map(opt => opt.value)
+
+    if (search) {
+      if (!newFilters.includes(searchScope)) newFilters.push(searchScope)
+      filtersToRemove = filtersToRemove.filter(val => val !== searchScope)
```

Comment:
This code is a bit ugly, but it sets up the filter selection array in a similar manner to the tax and country filter.  Right now the endpoints only seem to take a single parameter of a given type.  There is probably a simpler way of doing this with plain objects.  In the future, perhaps we would want to accommodate arrays of the same type of parameter.